hivemind/proto/averaging.proto

```diff
@@ -32,7 +25,6 @@ enum MessageCode {
 }
 
 message JoinRequest {
-  string endpoint = 1;          // A follower accepts incoming allreduce requests at this address
   bytes schema_hash = 2;        // A hash that describes follower's tensors (shapes, num tensors, etc)
```

Comment:
[the resulting protobuf starts from index=2; discussed with @borzunov and decided to keep it that way unless someone has strong objections]